pages/blog/hacktoberfest-2022.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Hacktoberfest was a blast!"
+date: 2022-11-28T10:00:00+01:00
+type: Community
+tags:
+  - Hacktoberfest
+cover: /img/posts/asyncapiXhacktoberfest.webp
+featured: true
+authors:
+  - name: Azeez Elegbede
+    photo: /img/avatars/ace.webp
+    link: https://twitter.com/_acebuild
+    byline: Preacher at AsyncAPI
+---
+
+Hacktoberfest was a blast this year! We can't believe it's over already. We had a great time participating with you all and wanted to share a bit of our experience in a review.
+
+## Why Hacktoberfest
+
+Hacktoberfest is an annual event that encourages people to contribute to open source throughout October. AsyncAPI is an open-source project and, like many other open-source projects, built and maintained by passionate people who often don't have the staff or budgets to do much more than keep the project alive. Hacktoberfest is all about giving back to those projects, sharpening skills, and celebrating all things open source, especially those who make open source so special.
+
+## Planting the seed
+
+For us, contributions come in all shapes and sizes. From typo fixes, bug reporting to more significant improvements. But that's not all! AsyncAPI also creates time to help onboard contributors that are finding it difficult to start contributing to the project or provide help to contributors that get stuck while contributing. 
+
+## How we prepared
+
+Not gonna lie, this year's Hacktoberfest snuck up on us like Batman in Gotham City, and as soon as we realized this, we immediately started planning. We kicked off by defining our goals for Hacktoberfest as we've always done. 
+
+- Promote [AsyncAPI Initiative](https://www.asyncapi.com/) as a place where we work not only on the AsyncAPI specification but also lots of tools
+- Help members of the broader open-source community make their first contributions in a friendly environment
+
+We spent time writing a how-to-contribute guide for folks participating for the first time and also wrote a maintainers guide for potential maintainers.
+
+AsyncAPI is a great project to contribute to because you can learn about AsyncAPI while fixing an issue. That's why we carefully curated issues and prepared especially for Hacktoberfest. And because most of these issues are existing issues, we made sure we reached out to project maintainers to identify which of them is a good one for Hacktoberfest contributors to look into
+
+## Was it worth it?
+
+Absolutely! Even despite the fact that we kicked off late, we still received an amazing contribution. AsyncAPI merged 23 PRs from 12 contributors as part of this year's Hacktoberfest! I know it's less than the previous years but let's look on the bright side because some of these contributions were fun to make.
```

Comment:
```suggestion
Absolutely! Even though we kicked off late, we still received an amazing contribution. AsyncAPI merged 23 PRs from 12 contributors as part of this year's Hacktoberfest! I know it's less than the previous years but let's look on the bright side because some of these contributions were fun to make.
```